core/tokens/colors.json

```diff
@@ -1,512 +1,418 @@
-          "rgba": { "r": 127, "g": 235, "b": 134, "a": 1 },
-          "cmyk": { "c": 46, "m": 0, "y": 43, "k": 8 }
+          "hex": "#31E88C",
+          "rgb": { "r": 49, "g": 232, "b": 140 },
+          "rgba": { "r": 49, "g": 232, "b": 140, "a": 1 }
         },
         {
           "value": 40,
           "name": "green-40",
-          "hex": "#44CF50",
-          "rgb": { "r": 68, "g": 207, "b": 80 },
-          "rgba": { "r": 68, "g": 207, "b": 80, "a": 1 },
-          "cmyk": { "c": 67, "m": 0, "y": 61, "k": 19 }
+          "hex": "#00CF64",
+          "rgb": { "r": 0, "g": 207, "b": 100 },
+          "rgba": { "r": 0, "g": 207, "b": 100, "a": 1 }
         },
         {
           "value": 50,
           "name": "green-50",
-          "hex": "#24AB31",
-          "rgb": { "r": 36, "g": 171, "b": 49 },
-          "rgba": { "r": 36, "g": 171, "b": 49, "a": 1 },
-          "cmyk": { "c": 79, "m": 0, "y": 71, "k": 33 }
+          "hex": "#00AB50",
+          "rgb": { "r": 0, "g": 171, "b": 80 },
+          "rgba": { "r": 0, "g": 171, "b": 80, "a": 1 }
         },
         {
           "value": 60,
           "name": "green-60",
-          "hex": "#1B8728",
-          "rgb": { "r": 27, "g": 135, "b": 40 },
-          "rgba": { "r": 27, "g": 135, "b": 40, "a": 1 },
-          "cmyk": { "c": 80, "m": 0, "y": 70, "k": 47 }
+          "hex": "#00853C",
+          "rgb": { "r": 0, "g": 133, "b": 60 },
+          "rgba": { "r": 0, "g": 133, "b": 60, "a": 1 }
         },
         {
           "value": 70,
           "name": "green-70",
-          "hex": "#196323",
-          "rgb": { "r": 25, "g": 99, "b": 35 },
-          "rgba": { "r": 25, "g": 99, "b": 35, "a": 1 },
-          "cmyk": { "c": 75, "m": 0, "y": 65, "k": 61 }
+          "hex": "#03612C",
+          "rgb": { "r": 3, "g": 97, "b": 44 },
+          "rgba": { "r": 3, "g": 97, "b": 44, "a": 1 }
         },
         {
           "value": 80,
           "name": "green-80",
-          "hex": "#17451E",
-          "rgb": { "r": 23, "g": 69, "b": 30 },
-          "rgba": { "r": 23, "g": 69, "b": 30, "a": 1 },
-          "cmyk": { "c": 67, "m": 0, "y": 57, "k": 73 }
+          "hex": "#08421F",
+          "rgb": { "r": 8, "g": 66, "b": 32 },
+          "rgba": { "r": 8, "g": 66, "b": 32, "a": 1 }
         },
         {
           "value": 90,
           "name": "green-90",
-          "hex": "#122B16",
-          "rgb": { "r": 18, "g": 43, "b": 22 },
-          "rgba": { "r": 18, "g": 43, "b": 22, "a": 1 },
-          "cmyk": { "c": 58, "m": 0, "y": 49, "k": 83 }
+          "hex": "#092E16",
+          "rgb": { "r": 9, "g": 46, "b": 22 },
+          "rgba": { "r": 9, "g": 46, "b": 22, "a": 1 }
         },
         {
-          "value": 100,
-          "name": "green-100",
-          "hex": "#0E1F11",
-          "rgb": { "r": 14, "g": 31, "b": 17 },
-          "rgba": { "r": 14, "g": 31, "b": 17, "a": 1 },
-          "cmyk": { "c": 55, "m": 0, "y": 45, "k": 88 }
+          "value": "95",
```

Comment:
`$md-gray-100` is also being used and would need to be mapped to `$md-gray-95`

Reply:
Fixed with new commit: updating core color variables